packages/ses/src/error/tame-console.js

```diff
@@ -53,21 +56,56 @@ export const tameConsole = (
   // utter that. That unnecessary shim forces the whole bundle into sloppy mode,
   // which in turn breaks SES's strict mode invariant.
 
+  /* eslint-disable @endo/no-polymorphic-call */
+
+  const reportReason = reason =>
+    causalConsole.warn('SES_UNHANDLED_REJECTION:', reason);
+
   // Node.js
   if (errorTrapping !== 'none' && globalThis.process !== undefined) {
-    // eslint-disable-next-line @endo/no-polymorphic-call
-    globalThis.process.on('uncaughtException', error => {
+    const sesDefaultPolicy = globalThis.Symbol.for('sesDefaultPolicy');
```

Comment:
I don't understand this yet, but why a symbol rather than a weakmap?

Reply:
Overly complex from a failed experiment.  I've simplified.